tools/monitoring/mysql/database.go

```diff
@@ -200,9 +199,22 @@ func (db *DB) IsPatternAlerting(errorPattern, jobPattern string, window time.Dur
 		WHERE ErrorPattern = ?
 		AND JobName REGEXP ?
 		AND TimeStamp > ?`, errorPattern, jobPattern, startTime)
-	if err := row.Scan(&nMatches, &nJobs, &nPRs); err != nil {
-		return false, err
-	}
 
-	return nMatches >= aTotal && nJobs >= aJobs && nPRs >= aPRs, nil
+	err := row.Scan(&nMatches, &nJobs, &nPRs)
+	return err == nil && nMatches >= aTotal && nJobs >= aJobs && nPRs >= aPRs, err
+}
+
+// execAffectingOneRow executes a given statement, expecting one row to be affected.
+func execAffectingOneRow(stmt *sql.Stmt, args ...interface{}) (sql.Result, error) {
+	r, err := stmt.Exec(args...)
+	if err != nil {
+		return r, fmt.Errorf("mysql: could not execute statement: %v", err)
```

Comment:
no need to prefix with mysql for all of these

Reply:
Updated to remove those extra messy error message.